src/_guides/language/effective-dart/usage.md

```diff
@@ -384,7 +391,7 @@ class UploadException {
 {% endprettify %}
 
 Assigning to a local variable can be cleaner and safer than using `!` every
-place the field or top-level variable is used:
+place the instance is used:
```

Comment:
**suggestion:** Clean up the language some to keep it concise.

```suggestion
Assign the value to a local scope variable. This reduces the chance for accidental errors and the length of the code.
```

Reply:
I think the original sentence is as concise as possible already, so I'm gonna skip this suggestion too.
- "Local scope variable" is redundant; "local" is intrinsic to scope. 
- "Accidental errors" also feels redundant; errors are typically unintentional. 

And again, some nuance is lost. "Length of code" is not the concern when it says using the `!` operator is not clean; if you compare the good and bad example, the length of the code is basically not affected at all. The "cleanness" concern here is, imo, about the excessive complexity of comprehending the null assertion operator so many times in a single statement.

I also don't think this line needs to be written in the imperative ("assigning" to "assign"). The instruction is already given on line 366: "_...assign the field's value to a local variable_". Line 393 is just commenting on that; basically saying "doing this is better than <bad example> because..."